server/documents/elements/label.html.eco

```diff
@@ -333,16 +353,89 @@ themes      : ['Default', 'GitHub']
   <div class="example" data-use-content="true" data-class="icon">
     <h4 class="ui header">Icon</h4>
     <p>A label can include an icon</p>
-    <a class="ui label">
+    <div class="ui label">
       <i class="mail icon"></i>
       Mail
-    </a>
```

Comment:
We could think of keeping this, but having a separate paragraph for it to show the backward compatible and special handling of `delete icon` and `close icon` within a label, when only `ui label` (instead of `ui right icon label`) is used